charts/common/values.yaml

```diff
@@ -1,58 +1,125 @@
-# app: the-app
-# shortname: theapp
-# team: yourteam
-# env: dev
+# -- Application name, typically on the form `the-application`
```

Comment:
Since values.yaml is the default values file, is it possible to create a seperate values file for examples and readme generation?

Reply:
Did not get it to work. Also think it makes sense to keep doc and implementation together. A new stanza will have a higher chance of being documented when its derived from the default values file IMHO.